packages/types/src/context.ts

```diff
@@ -124,3 +126,8 @@ export interface MissingInstrumentationContext extends Record<string, unknown> {
   package: string;
   ['javascript.is_cjs']?: boolean;
 }
+
+export interface FeatureFlagContext extends Record<string, unknown> {
+  // This should only be modified by @sentry/util methods (insertToFlagBuffer).
+  readonly values: FeatureFlag[];
```

Comment:
This readonly is a little unnecessary but verified it works, the buffer contents can still be modified. I'll prob remove in a future commit